networks/ligand_networks_for_developers.ipynb

```diff
@@ -0,0 +1,895 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "mappings[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0411304f",
+   "metadata": {},
+   "source": [
+    "## Creating a custom network planner\n",
+    "\n",
+    "A network planner is a function that uses a mapper on pairs of ligands, and returns the `Network` object. The `Network` can be created for a list of `LigandAtomMapping`s. If you have additional nodes that are not in a mapping, you can add those with the `nodes` keyword.\n",
+    "\n",
+    "First let's look at an overly simple network planner. It takes in the ligands and at least one mapper, and returns a `Network`. Sometimes the ligands need to be organized into different variables -- here we need to separate ligands with 3 heavy atoms from those with 2; in a star map, you need to identify which ligand is the central hub. Because of this, the exact call signature of the network planner can differ, but it should always return a `Network`."
```

Comment:
> Sometimes the ligands need to be organized into different variables -- here we need to separate ligands with 3 heavy atoms from those with 2; in a star map, you need to identify which ligand is the central hub. Because of this, the exact call signature of the network planner can differ, but it should always return a `Network`.

I know what you're saying here, but somehow it's not reading super well, I can't point out to what the problem is though.